plugins/api/src/main/kotlin/PluginFactory.kt

```diff
@@ -21,6 +21,12 @@ package org.ossreviewtoolkit.plugins.api
 
 import org.ossreviewtoolkit.utils.common.getLoaderFor
 
+/**
+ * IDEA:
```

Comment:
Consider using our more commonly used `TODO: If a plugin ...` comment style for proper highlighting in the IDE (even if if really maybe is more an idea than a to-do).

Reply:
That was a note I just forgot to remove.